NitroxPatcher/Patches/Dynamic/Planter_IsAllowedToAdd_Patch.cs

```diff
@@ -0,0 +1,30 @@
+using System.Reflection;
+using HarmonyLib;
+using NitroxClient.GameLogic;
+using NitroxClient.MonoBehaviours;
+using NitroxModel.DataStructures;
+using NitroxModel.Helper;
+
+namespace NitroxPatcher.Patches.Dynamic;
+
+public class Planter_IsAllowedToAdd_Patch : NitroxPatch, IDynamicPatch
+{
+    public static readonly MethodInfo TARGET_METHOD = Reflect.Method((Planter p) => p.IsAllowedToAdd(default(Pickupable), default(bool)));
```

Comment:
```suggestion
    public static readonly MethodInfo TARGET_METHOD = Reflect.Method((Planter p) => p.IsAllowedToAdd(default, default));
```

Reply:
We should keep those explicit types, if you use infering type, compilation will still work even there's a minor signature refactor